server/routerlicious/packages/routerlicious-base/src/riddler/tenantManager.ts

```diff
@@ -236,6 +247,16 @@ export class TenantManager {
      * Retrieves the secret for the given tenant
      */
     public async getTenantKeys(tenantId: string, includeDisabledTenant = false): Promise<ITenantKeys> {
+        const lumberProperties = { [BaseTelemetryProperties.tenantId]: tenantId };
+
+        // Read from cache first
+        const cachedKey = await this.cache.get(tenantId);
+        if (cachedKey) {
+            Lumberjack.info(`Tenant key(s) found in cache`, lumberProperties);
+            return this.decryptCachedKeys(cachedKey);
+        }
+
+        // Read from database if keys aren't found in the cache
```

Comment:
It might be useful to add a log line here so that we can get the cache hit rate and also to get a sense of how often we are calling the db for retrieving the keys.

Reply:
Added a log for when we read from the database when keys aren't found in the cache